components/raftstore/src/store/worker/read.rs

```diff
@@ -692,6 +713,7 @@ where
             // Forward to raftstore.
             Ok(None) => self.redirect(RaftCommand::new(req, cb)),
             Err(e) => {
+                println!("{:?}", e);
```

Comment:
Please remove this verbose print.

Reply:
I forget to remove it. Thanks for the reminding.